src/dotnet/commands/dotnet-migrate/Program.cs

```diff
@@ -21,29 +21,50 @@ public static int Run(string[] args)
 
             CommandLineApplication app = new CommandLineApplication();
             app.Name = "dotnet migrate";
-            app.FullName = ".NET Migrate Command";
-            app.Description = "Command used to migrate project.json projects to msbuild";
+            app.FullName = LocalizableStrings.AppFullName;
+            app.Description = LocalizableStrings.AppDescription;
             app.HandleResponseFiles = true;
             app.HelpOption("-h|--help");
 
-            CommandArgument projectArgument = app.Argument("<PROJECT_JSON/GLOBAL_JSON/PROJECT_DIR>",
+            CommandArgument projectArgument = app.Argument(
+                $"<{LocalizableStrings.CmdProjectArgument}>",
                 string.Join(Environment.NewLine,
-                "The path to ",
-                " - a project.json file to migrate.",
-                "or",
-                " - a global.json file, it will migrate the folders specified in global.json.",
-                "or",
-                " - a directory to migrate, it will recursively search for project.json files to migrate.",
-                "Defaults to current directory if nothing is specified."));
+                LocalizableStrings.CmdProjectPath,
```

Comment:
This feels wrong to me. We shouldn't have multiple localization entries for a single string just to maintain formatting. Can't we use an "@" at the localized string to maintain formatting there?